app/src/main/java/com/puzzlebench/clean_marvel_kotlin/presentation/mvp/CharacterPresenter.kt

```diff
@@ -5,19 +5,30 @@ import com.puzzlebench.clean_marvel_kotlin.presentation.base.Presenter
 import io.reactivex.android.schedulers.AndroidSchedulers
 import io.reactivex.disposables.CompositeDisposable
 import io.reactivex.schedulers.Schedulers
+import io.realm.Realm
 
-class CharacterPresenter(view: CharacterView, private val getCharacterServiceUseCase: GetCharacterServiceUseCase, val subscriptions: CompositeDisposable) : Presenter<CharacterView>(view) {
+
+class CharacterPresenter(view: CharacterView, val model: CharacterModel, private val getCharacterServiceUseCase:
```

Comment:
same, the use case shoudnt be here,it should be inside the model. Also you can remove the "subscriptions"

Reply:
Done